src/test/java/com/emc/mongoose/controls/ConfigServletTest.java

```diff
@@ -0,0 +1,79 @@
+package com.emc.mongoose.controls;
+
+import com.emc.mongoose.control.ConfigServlet;
+import com.github.akurilov.confuse.Config;
+import com.github.akurilov.confuse.impl.BasicConfig;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ @author veronika K. on 02.11.18 */
+public class ConfigServletTest {
+
+	private static final int PORT = 9999;
```

Comment:
this value should be obtained from the config

Reply:
But it is Unit test, there is no mongoose

Reply:
I can change port in system test (ControlApiTest)